src/components/Sidebar/LegalLinks.jsx

```diff
@@ -51,6 +53,9 @@ const LegalLinks = (props: Props) => {
       <GnoButtonLink className={classes.buttonLink} onClick={openCookiesHandler}>
         Cookies
       </GnoButtonLink>
+      <Link className={classes.link} to="https://github.com/gnosis/safe-react/releases" target="_blank">
```

Comment:
Maybe you can hide this if the env variable is not defined, otherwise it will look like this:
![image](https://user-images.githubusercontent.com/21086218/71026294-b38b3380-20e7-11ea-909b-083b865b5998.png)